app/helpers/application_helper.rb

```diff
@@ -102,4 +102,24 @@ def finders_sorted_by_title
     finders.sort_by {|_, value| value[:title] }
   end
 
+  def publish_text(manual, slug_unique)
+    if manual.state == "published"
+      text = "<p>There are no changes to publish.</p>"
+    elsif manual.state == "withdrawn"
+      text = "<p>The document is withdrawn. You need to create a new draft before it can be published.</p>"
```

Comment:
Oh, yes.  I copied this almost directly from specialist-publisher but not with a critical enough eye.